textwrap AttributeError: 'NoneType' object has no attribute 'expandtabs'

```
$ python -c "import pyvista;print(pyvista.Report())"
  
Traceback (most recent call last):
  File "<string>", line 1, in <module>
  File "/opt/hostedtoolcache/Python/3.10.6/x64/lib/python3.10/site-packages/scooby/report.py", line 298, in __repr__
    for txt in textwrap.wrap(self._mkl_info, self.text_width - 4):
  File "/opt/hostedtoolcache/Python/3.10.6/x64/lib/python3.10/textwrap.py", line 387, in wrap
    return w.wrap(text)
  File "/opt/hostedtoolcache/Python/3.10.6/x64/lib/python3.10/textwrap.py", line 359, in wrap
    chunks = self._split_chunks(text)
  File "/opt/hostedtoolcache/Python/3.10.6/x64/lib/python3.10/textwrap.py", line 345, in _split_chunks
    text = self._munge_whitespace(text)
  File "/opt/hostedtoolcache/Python/3.10.6/x64/lib/python3.10/textwrap.py", line 154, in _munge_whitespace
    text = text.expandtabs(self.tabsize)
AttributeError: 'NoneType' object has no attribute 'expandtabs'
Error: Process completed with exit code 1.
```

--- a/scooby/report.py
+++ b/scooby/report.py
@@ -102,7 +102,7 @@ class PlatformInfo:
             elif numexpr:
                 self._mkl_info = numexpr.get_vml_version()
             else:
-                self._mkl_info = 'unknown'
+                self._mkl_info = None
 
         return self._mkl_info
 
@@ -293,9 +293,9 @@ class Report(PlatformInfo, PythonInfo):
             text += f'{name:>{row_width}} : {version}\n'
 
         # MKL details
-        if self.mkl_info != 'unknown':
+        if self.mkl_info:
             text += '\n'
-            for txt in textwrap.wrap(self._mkl_info, self.text_width - 4):
+            for txt in textwrap.wrap(self.mkl_info, self.text_width - 4):
                 text += '  ' + txt + '\n'
 
         # Finish
@@ -377,7 +377,7 @@ class Report(PlatformInfo, PythonInfo):
         html += "  </tr>\n"
 
         # MKL details
-        if self.mkl_info != 'unknown':
+        if self.mkl_info:
             html = colspan(html, self.mkl_info, self.ncol, 2)
 
         # Finish
@@ -413,7 +413,7 @@ class Report(PlatformInfo, PythonInfo):
             out[name] = version
 
         # MKL details
-        if self.mkl_info != 'unknown':
+        if self.mkl_info:
             out['MKL'] = self.mkl_info
 
         return out

